src/Driver/PDO/DriverAndPDOException.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Driver\PDO;
+
+use Doctrine\DBAL\Driver\Exception as DriverException;
+use PDOException;
+
+/**
+ * @internal
+ *
+ * @psalm-immutable
+ */
+final class DriverAndPDOException extends PDOException implements DriverException
```

Comment:
Shall we just call it `PDOException`?

Reply:
… one that is resolvable by not importing the upstream class and referencing it via FQCN instead.

Reply:
This will cause changes to exec, prepare, query, lastInsertId. Its OK?

Reply:
It will? Why?

Reply:
Renamed to `PDOException`